New blog post

RonaldVisser opened the pull request:
Hi @steffilazerte, 
Here is my blog-post. I hope that this is fitting for rOpenSci

steffilazerte commented:
Hi @RonaldVisser, great!

I'll take a look next week. In the meanwhile, could you just add this checklist to your PR and double check that it's good to go? 

This is based on the checklist in our [blog guide](https://blogguide.ropensci.org/editorchecklistpeer.html), but is a bit more up-to-date. Here's an example of one of my [posts](https://github.com/ropensci/roweb3/pull/587) using a similar checklist, for context.

Thanks!

```
* [ ] I have read the Content Guidelines.
* [ ] I have read the Technical Guidelines.
* [ ] I used or followed the R Markdown or Markdown template.
* [ ] I have followed the Style Guide.
* [ ] I created or updated my author metadata with correct folder name.
* [ ] I have added relevant tags after browsing existing tags (including "community" tag).
* [ ] I have added the "tech notes" tag if this is a technote.
* [ ] I ran `roblog::ro_lint_md()` on index.md (optional).
* [ ] I ran `roblog::ro_check_urls()` on index.md (optional).
* [ ] I ran a spell-check on index.md.
* [ ] I have added the tags - Software Peer Review, my-packagename.
* [ ] I have added the package-version YAML tag.
* [ ] I have added acknowledgement of the reviewers' work (with links to reviewers).
* [ ] I have added a link to the software peer review thread.
 ```

RonaldVisser commented:
Thanks @steffilazerte ! This is my first blog post, so I hope it is okay. Here is the checklist:

* [x] I have read the Content Guidelines.
* [x] I have read the Technical Guidelines.
* [x] I used or followed the R Markdown or Markdown template.
* [x] I have followed the Style Guide.
* [x] I created or updated my author metadata with correct folder name.
* [x] I have added relevant tags after browsing existing tags (including "community" tag).
* [ ] I have added the "tech notes" tag if this is a technote.
* [x] I ran `roblog::ro_lint_md()` on index.md (optional).
* [x] I ran `roblog::ro_check_urls()` on index.md (optional).
* [x] I ran a spell-check on index.md.
* [ ] I have added the tags - Software Peer Review, my-packagename.
* [x] I have added the package-version YAML tag.
* [x] I have added acknowledgement of the reviewers' work (with links to reviewers).
* [ ] I have added a link to the software peer review thread.

Just a few question/remarks:

- I have added relevant tags, but if you have suggestions for tags to add, please let me know. Forgot to add my packagename to the tags
- the reviewers are acknowledged in the post, but the link to the peer review (https://github.com/ropensci/software-review/issues/627) is missing. I can add that after you have read the post

steffilazerte reviewed (commented):
Hi @RonaldVisser,

This is great! A really nice background to how you came about writing dendroNetwork. I feel like a lot of people have had a similar experience, of first making the scripts and then later turning it into a package. It's a nice set of advice for people thinking about maybe tackling their first package.

Also, how cool is it that you can trace the wood in old Roman ships to where it came from!

I have made some suggestions to the PR. See what you think, feel free to accept my suggestions (you can commit them directly to the PR), modify them, or reject them (just ignore them). They are only wording suggestions, so it's up to you if you would like to make any changes. However, if you could mark the comments as "Resolved" once you've looked them over, that will help me make sure you did get a chance to review them (GitHub sometimes collapses suggestions in the middle, so they can be semi-hidden and get missed). 

And finally, we're good to publish when ever you've had a chance to review the comments. Depending on how busy you are, we could do this Thursday (June 6th) (if you can review the comments by Wednesday) or next Tuesday (June 11th) (if you can review the comments by Monday). No worries if you're not sure when you'll have a chance to review, we can set it for a later date too. Just remember to change the date in the YAML and in the folder name when you think you have a date that will work!

Once you've had a chance to review and make any changes, ping me and I'll make sure we're good to go. The day of publication, I'll merge the PR and we'll be off! I'll be sharing the post on Mastodon and LinkedIn, so will share a preview of that text here before hand so you can comment.

Thanks!

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [ ] publication date is ok -> **We just need to decide on a date**
* [X] alternative text of images is informative
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on [index.md](http://index.md/)
* [X] I ran `roblog::ro_check_urls()` on [index.md](http://index.md/)
* [X] I ran a spell-check on [index.md](http://index.md/)
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary
* [X] YAML package-version included
* [X] YAML subject tags - software peer review, packagename
* [X] acknowledges and links to reviewers
* [X] links to peer review thread

RonaldVisser commented on content/blog/2024-05-31-from-scripts-to-package/index.md:
```diff
@@ -0,0 +1,53 @@
+package_version: 0.5.4
+author:
+  - Ronald Visser
+date: 2024-05-30
```
6th of june is fine

RonaldVisser commented:
Thank you for going through the text and thank you for the corrections and suggestions @steffilazerte . I accepted all. And publication the 6th of June is fine. Do I need to change the folder name too?

And yes, it is great that tree-rings can not only be used for dating wood, but also to estimate the provenance of wood. 

Thank you for all your help :)

steffilazerte commented:
Hi @RonaldVisser, super speedy! Yes we do need to adjust the date of the folder, but since I also needed to add myself as editor, I just did both at once. So now we're good to go!

Here is the draft post for Mastodon and LinkedIn, any comments/suggestions (tags etc.?)

```
[blog] From scripts to package. Developing dendroNetwork and learning with rOpenSci 🌲

@RonaldVisser@akademienl.social / Ronald Visser shares the story of going from a set of scripts for comparing tree rings to a fully reviewed R package! 🎉
Ronald also shares some resources and advice for those thinking about starting their first R package 📖
Such a small thing, yet it led to tracing wood used to build Roman ships found in the River Rhine! ⚓

https://ropensci.org/blog/2024/06/06/from-scripts-to-package/

#RStats

🙏 And of course, big thanks to rOpenSci reviewers @kaija_bean@ecoevo.social / Kaija Gahm and Zach Gajewski!
```

steffilazerte commented:
Great thanks @RonaldVisser!
